specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -226,9 +226,65 @@
         "daily",
         "hourly",
         "minutely",
-        "secondly"
+        "secondly",
+        "microsecond"
```

Comment:
You also need to add `microsecond` to the `x-ms-enum` list for this enum.

Reply:
updated.